internal/pkg/cli/app_show.go

```diff
@@ -268,6 +300,14 @@ func BuildAppShowCmd() *cobra.Command {
 			o.describer = d
 			return nil
 		},
+		initWorkSpace: func(o *showAppOpts) error {
```

Comment:
I think I did a poor job in explaining why I added an `initDescriber` method 😛.
I don't think we need this method in the struct so the workspace can be initialized in `PreRunE`. The difference between the two is that `initDescriber` can only initialize its client **after** the flag values are asked:
```go
describe.NewWebAppDescriber(o.ProjectName(), o.appName)
```
On the other hand, the workspace doesn't need any parameters to get initialized so it can be done in `PreRunE`.

Reply:
Understood:+1: